### PR TITLE
New runner: net finder

### PR DESCRIPTION
**Pull request by mirceaulinic**
### What does this PR do?

A new runner: `net` to search network related details easy and fast.

### CODE REVIEW

**Comment by mirceaulinic** [comment]
I have a lint error: https://jenkins.saltstack.com/job/PR/job/salt-pr-lint-n/8008/violations/file/salt/runners/net.py/

What I'm trying to do there is forcing the usage of a certain outputter: https://docs.saltstack.com/en/develop/ref/output/all/salt.output.table_out.html

That print works fine (at least in py2), the linter suggested to avoid using parenthesis  (```print(ret)```), but here it complains about `modernize`.
Could you please point some suggestions to improve this?

File: salt/runners/net.py [hunk 1]
```diff
@@ -0,0 +1,922 @@
...
+              - lo0
+              - em1
+              - jsrv
+              - fxp0
+            outputter: yaml
+
+.. versionadded:: Nitrogen
+'''
+
+from __future__ import absolute_import
```
Comment (cachedout) [inline salt/runners/net.py]: `from __future__ import print_function`

File: salt/runners/net.py [hunk 2]
```diff
@@ -0,0 +1,922 @@
...
+                                         'table',
+                                         __opts__,
+                                         title=title,
+                                         rows_key='rows',
+                                         labels_key='labels')
+        else:
+            ret = salt.output.out_format(rows,
+                                         net_runner_opts.get('outputter'),
+                                         __opts__)
+        print ret
```
Comment (cachedout) [inline salt/runners/net.py]: `print(ret)`

File: salt/runners/net.py [hunk 3]
```diff
@@ -0,0 +1,922 @@
...
+        ARP Entries for IP 10.10.10.7
+        _____________________________________________________________________________
+        |    Device    |     Interface     |        MAC        |     IP     |  Age  |
+        _____________________________________________________________________________
+        | edge01.flw01 | irb.349 [ae0.349] | 2C:60:0C:2A:4C:0A | 10.10.10.7 | 832.0 |
+        _____________________________________________________________________________
+    '''
+    if not addr:
+        if display:
+            print 'Please type a valid MAC/IP Address / Device / Interface / VLAN'
```
Comment (cachedout) [inline salt/runners/net.py]: print('Please type a valid MAC/IP Address / Device / Interface / VLAN')

**Review by cachedout — CHANGES_REQUESTED** [review]
Hi @mirceaulinic 

I have made suggestions in-line that should clear up the lint errors you are seeing.

File: salt/runners/net.py (hunk 1 above)
Comment (mirceaulinic) [inline salt/runners/net.py]: Oh, yes! Totally forgot about that! Thank you @cachedout!

**Comment by mirceaulinic** [comment]
I have updated this branch @cachedout

**Comment by cachedout** [comment]
@mirceaulinic Thank you! When the tests finish, I will hopefully get this in.

**Comment by mirceaulinic** [comment]
No problem, let me know whenever something doesn't look about right!

**Pull request merged**